packages/runtime/src/helpers/blobStorage.ts

```diff
@@ -0,0 +1,34 @@
+import type { Blobs } from '@netlify/blobs/dist/src/main'
+
+let blobs: Promise<Blobs>
+
+export const getBlobStorage = async ({
+  apiHost,
+  token,
+  siteID,
+  deployId,
+}: {
+  apiHost: string
+  token: string
+  siteID: string
+  deployId: string
+}): Promise<Blobs> => {
+  // eslint-disable-next-line no-new-func
```

Comment:
This is a workaround to allow us to use `@netlify/blobs` which is currently ESM only. Hat tip to @pieh for the hack.